drivers/clock_control/clock_control_nrf2_fll16m.c

```diff
@@ -64,7 +64,7 @@ static const struct clock_options {
 struct fll16m_dev_data {
 	STRUCT_CLOCK_CONFIG(fll16m, ARRAY_SIZE(clock_options)) clk_cfg;
 	struct onoff_client hfxo_cli;
-	struct clock_lrcconf_sink lrcconf_sink;
+	sys_snode_t lrcconf_client_node;
```

Comment:
nit: `fll16m_node` might be a bit less generic?

Reply:
Sounds good - fixed for `hfxo` module as well